lib/setup.coffee

```diff
@@ -114,7 +114,7 @@ module.exports = (app) ->
       dest: path.resolve(__dirname, "../public")
     app.use require("browserify-dev-middleware")
       src: path.resolve(__dirname, "../")
-      transforms: [require("jadeify"), require('caching-coffeeify')]
+      transforms: [require("jadeify"), require('caching-coffeeify'), require('babelify')]
```

Comment:
Moving jadeify to the end might work around the `require` thing I mentioned above.